repos/fdbt-site/tests/pages/products/otherProducts.test.tsx

```diff
@@ -0,0 +1,210 @@
+import { shallow } from 'enzyme';
```

Comment:
Tests for OtherProducts Table?

Reply:
Moved it into the same file, as its not a separate component for the p2p products stuff.